builtins.go

```diff
@@ -968,6 +969,56 @@ func builtinStrReplace(i *interpreter, trace traceElement, strv, fromv, tov valu
 	return makeValueString(strings.Replace(sStr, sFrom, sTo, -1)), nil
 }
 
+func base64DecodeGoBytes(i *interpreter, trace traceElement, str string) ([]byte, error) {
+	strLen := len(str)
+	if strLen%4 != 0 {
+		msg := fmt.Sprintf("input string appears not to be a base64 encoded string. Wrong length found (%d)", strLen)
+		return nil, makeRuntimeError(msg, i.getCurrentStackTrace(trace))
+	}
+
+	decodedBytes, err := base64.StdEncoding.DecodeString(str)
+	if err != nil {
+		return nil, i.Error(fmt.Sprintf("failed to decode: %s", err), trace)
+	}
+
+	return decodedBytes, nil
+}
+
+func builtinBase64DecodeBytes(i *interpreter, trace traceElement, input value) (value, error) {
+	vStr, err := i.getString(input, trace)
+	if err != nil {
+		msg := fmt.Sprintf("base64DecodeBytes requires a string, got %s", input.getType().name)
+		return nil, makeRuntimeError(msg, i.getCurrentStackTrace(trace))
+	}
+
+	decodedBytes, err := base64DecodeGoBytes(i, trace, vStr.getGoString())
+	if err != nil {
+		return nil, i.Error(fmt.Sprintf("failed to base64 decode: %v", err.Error()), trace)
```

Comment:
This results in "RUNTIME ERROR: failed to base64 decode: RUNTIME ERROR: something something". Wrapping one runtime error in another is not ideal.  It's better to just pass through err here.

Reply:
fixed